sound/soc/intel/boards/cml_rt1011_rt5682.c

```diff
@@ -488,15 +506,23 @@ static struct snd_soc_codec_conf rt1011_conf[] = {
 		.dlc = COMP_CODEC_CONF("i2c-10EC1011:01"),
 		.name_prefix = "WR",
 	},
+	{
+		.dlc = COMP_CODEC_CONF("i2c-10EC1011:02"),
+		.name_prefix = "TL",
+	},
+	{
+		.dlc = COMP_CODEC_CONF("i2c-10EC1011:03"),
+		.name_prefix = "TR",
+	},
```

Comment:
We don't need a separate codec_conf[].  rt1011_conf_4spk[] is an extension of rt1011_conf_2spk[]. Adding "TL" and "TR" into codec_conf[] is enough.

Reply:
that's good point, I added TL/TR to rt1011_conf. num_config is set to 2 initially, probe will update it to 4 when 4 speakers are supported.